tests/demo/molecule/playbook.retry

```diff
@@ -0,0 +1,2 @@
+demo-01-centos7-ubuntu
```

Comment:
Oops look like this file got committed.  We should add a .gitignore for *.retry.